README.md

```diff
@@ -210,170 +198,208 @@ Having done this setup, this is what the development cycle looks like moving for
 2. Rebuild specific package
 3. Rebuild client application.
 
-### Building changes to Rust source
+### Building Rust Components
 
-If changes are made to `packages/wasm` see [here](./packages/wasm/README.md) for info on building from source.
+For changes to WebAssembly components in `packages/wasm`, refer to the [WebAssembly build guide](./packages/wasm/README.md).
 
-## Publishing
+## Publishing New Versions
 
-You must have at least nodejs v18 to do this.
+Prerequisites:
 
-1. Install the latest packages with `yarn install`
+- Node.js v18.0.0 or later
 
-2. Run `yarn bump` to bump the version
+Publishing steps:
 
-3. Build all the packages with `yarn build`
+1. Update dependencies: `yarn install`
+2. Increment version: `yarn bump`
+3. Build packages: `yarn build`
+4. Run tests:
+   - Unit tests: `yarn test:unit`
+   - E2E tests: `yarn test:local`
+5. Generate documentation: `yarn gen:docs --push`
+6. Publish packages: `yarn publish:packages`
+7. Commit with message: "Published version X.X.X"
```

Comment:
Add a previous step to create a release PR like this one https://github.com/LIT-Protocol/js-sdk/pull/753
That is:
- branching from `master` and requesting to merge back to it
- calling the branch `vX.X.X-Publish`
- setting the changelog as the PR description
- adding the `Release` tag in the PR